contracts/staking/fuse/FuseStakingV4.sol

```diff
@@ -217,7 +265,22 @@ contract FuseStakingV4 is Initializable, OwnableUpgradeable, DSMath {
     uint256 amount
   ) internal virtual {
 		_withdraw(from, address(this), amount);
-		_stake(to, address(0), amount);
+		uint256 givebackRatio = getTransferGivebackRatio(to, from);
+		_stake(to, address(0), amount, givebackRatio);
+	}
+
+	/**
+	 * @dev determines the giveback ratio of a transferred stake
+	 * @param to the receiver
+	 * @param from the sender
+	 * @return receiver average giveback ratio if he has one, otherwise sender giveback ratio
+	 */
+	function getTransferGivebackRatio(address to, address from) internal view returns (uint256){
+		return stakersGivebacks[to].avgRatio > 0 ?
+					stakersGivebacks[to].avgRatio :
+					stakersGivebacks[from].avgRatio > 0 ?
+						stakersGivebacks[from].avgRatio :
+						DEFAULT_GIVEBACK_RATIO;
```

Comment:
Generally there shouldn't be a situation where the sender doesn't have a giveback ratio and we shouldn't get to this line. Can keep as is or throw